src/FrameRipper.py

```diff
@@ -0,0 +1,79 @@
+import cv2
+import numpy as np
+import os 
+
+#Frame Ripper Script WIP
+#Working on adding subset functionality in order to randomly grab sets of frames instead of all of them
+#
+#Steve
```

Comment:
I guess we can go with what Nick is doing and add the python author
```suggestion
__author__ = 'Steve'
```